src/components/Login.vue

```diff
@@ -5,41 +5,129 @@
         <p>Loading...</p>
       </div>
     </transition>
-    <section>
-      <div class="col1">
-        <h1>CoursePlan</h1>
-        <p>
-          Welcome to CoursePlan, your destination for creating plans surrounding your curriculum and
-          requirements
-        </p>
-      </div>
-      <div class="col2">
-        <form @submit.prevent>
-          <h1>Welcome Back</h1>
 
-          <div class="login-container">
-            <button @click="socialLogin" class="social-button">
-              <img alt="Google Logo" src="../assets/google-logo.png" />
-            </button>
 
-            <div class="login-description">Continue With Google Sign-In</div>
-          </div>
+      <div class="landing">
+        <!--TOP BAR-->
+        <div class="container p-0">
+            <div class="row top-bar justify-content-between no-gutters">
+                <div class="col-10">
+                    <img class="logo" src="../assets/images/logo.svg" alt = "logo" />
+                  </div>
+            </div>
+        </div>
 
-          <!-- <label for="email1">Email</label>
-          <input v-model.trim="loginForm.email" type="text" placeholder="you@email.com" id="email1" />
+        <!--PLAN AHEAD-->
+        <div class="container p-0">
+            <div class="row top-bar no-gutters">
+                <div class="col">
+                    <div class="row no-gutters"><h1 class="plan-head mt-5">Plan Your Courses Ahead</h1></div>
+                    <div class="row no-gutters"><p class="plan-subhead">Introducing the new and easiest way to plan courses at Cornell</p></div>
+                    <div class="row">
+                        <div class="col-5" >
+                            <button  @click="socialLogin"  class="email-button" variant= "primary"  v-on:click="addUser"> GET STARTED </button>
+                        </div>
+                    </div>
+                </div>
+                <div class="col">
+                    <img id= "hide"  style="position:relative" class="laptop" src="../assets/images/laptop.png" alt = "laptop preview" />
```

Comment:
It can be good practice to use the '@' symbol in Vue instead of the '..' in case the template files ever get moved, and you have to navigate differently to the src folder. Not a big deal but could be a quick fix